deploy/cr.yaml

```diff
@@ -280,6 +280,8 @@ spec:
     net:
       port: 27017
       hostPort: 0
+#     tls:
+#       mode: preferSSL
```

Comment:
We want to deprecate the whole spec.mongod section.
Also preferSSL is not only for mongod.
What about changing this flag in the custom configuration?